aiohttp_admin/security.py

```diff
@@ -71,16 +71,42 @@ def __init__(self, username, password, permissions=None):
 
 class DummyTokenIdentityPolicy(AbstractIdentityPolicy):
 
+    def __init__(self, server_secret=None):
+        '''
+            Makes identity tokens using HMAC(SHA-512) over a
+            server-side secret.
+
+            Provide a secret (20+ bytes) or we'll pick one
+            at runtime.
+        '''
+        from hmac import HMAC
```

Comment:
HMAC is [built in](https://docs.python.org/3/library/hmac.html) but yes, maybe this belongs in `aiohttp_security`